internal/node/node.bzl

```diff
@@ -200,7 +200,7 @@ def _nodejs_binary_impl(ctx):
     node_tool_files.append(ctx.file._bazel_require_script)
 
     if not ctx.outputs.templated_args_file:
-        templated_args = ctx.attr.templated_args
+        templated_args = [a for a in ctx.attr.templated_args]
```

Comment:
this string list is frozen, so I created a new list so we can mutate it (add the pnp flag if necessary)